.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+const config = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: ['airbnb-base'],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/TheCodeCartel/mergify-test-service/pullRequest?prid=7838955)
```suggestion
  extends: ["airbnb-base"],
```